packages/css/src/radio-button/radio-button.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../node_modules/@utrecht/components/radio-button/css";
+
+// Similar concept as Utrecht to set private variables and update them when specific state is set.
+.amsterdam-radio-button {
+  --_amsterdam-radio-button-background-color: var(--amsterdam-radio-button-background-color, currentColor);
+}
+
+.amsterdam-radio-button:checked {
+  background-image: radial-gradient(
+    circle,
+    var(--_amsterdam-radio-button-background-color, transparent) (55% - 5%),
+    currentColor 60%
```

Comment:
I think the styling is off,  a checked radio should have a blue border, then a white border, then a blue fill. It's completely filled in the current implementation (on my machine at least)